test/e2e/helper/alcotest_test_helper.ml

```diff
@@ -0,0 +1,7 @@
+let rec wakeup_until_resolved p =
```

Comment:
It should be explained somewhere that to make the tests work with js we use a custom lwt event loop. 
Either in [CONTRIBUTING.md](https://github.com/mirage/alcotest/blob/main/CONTRIBUTING.md#expect-tests) or in `alcotest_test_helper.ml`.